runner/run.go

```diff
@@ -20,13 +21,13 @@ import (
 
 var errBadTestRequest = errors.New("ftw/run: bad test: choose between data, encoded_request, or raw_request")
 
-// Run runs your tests with the specified Config.
-func Run(tests []test.FTWTest, c Config, out *output.Output) (*TestRunContext, error) {
+// Run runs your tests with the specified Config. Returns error if some test failed
```

Comment:
That part of the comment was removed on purpose.
```suggestion
// Run runs your tests with the specified Config.
```